contents/handbook/engineering/feature-pricing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Pricing principles
+sidebar: Handbook
+showTitle: true
+---
+
+## In an ideal world, Posthog’s pricing enables users and organizations to:
+
+- Use PostHog for free if they are hobbyists or pre-PMF
+- Consider PostHog as a candidate amongst the array of alternate options
+- Experience the product before paying for it
+- Start paying when they are ready, on their own, with few hurdles
+- Transparently pay for the value they receive
+  - Eg. Usage-based pricing on events, recordings
+  - Eg. Paying per product so they only pay for what they use
```

Comment:
I think we want a principle about grandfathering in old customers

Reply:
I personally hesitate to add that as a principle since sometimes it makes sense to not grandfather people. I view these things as "hills we are willing to die on." However, if grandfathering is a "hill you are willing to die on" then 1) that is very good info for us to know, and 2) we can add it. Otherwise I'd maybe put it as a recommendation somewhere else, to grandfather when it's reasonable.

Reply:
I put a note about this below but lmk if you think it should be a Principle.

Reply:
From my convos with @jamesefhawkins it seems to be in the range of "hill you are willing to die on unless you have a very strong reason otherwise". So 95%+ of pricing changes should be like this. Maybe in future we'd change the tone but particularly for the current stage the goodwill is very important

Reply:
"- When possible, we should try to grandfather existing customers' pricing tiers to avoid unexpected pricing changes. However, this does not mean that they should get all new features, only that they get to continue using their existing features at the price they originally agreed to. Upgrading to a "current" plan to get new features should be easy."

I like this idea but I'd lean more towards being generous at this stage and if they are on a paying plan for those set of features then they'd get access to them. e.g. if they are paying for product analytics but not for session recording, then they can't use the new paid session recording features after we add a paid plan. But if they are paying a lower rate for product analytics and we raise the price for new users, then they still get the new product analytics features

Reply:
But I don't feel super strongly on this. Worth checking with James/Tim and then up to make the decision

Reply:
Yeah I think if we are charging by product then your suggestion makes more sense. I was thinking about this in more of a traditional way where you charge for the whole platform and the customers who aren't paying much get a ton of new features/products for free. But by charging by product this is alleviated. Will update.

I've decided not to make this a principle. I see the principles as informing us in how to price our products, and grandfathering isn't about _how to price_, really, it's more about how to care for customers in the real world. So I am going to keep this in the "In the real world" section.